.github/workflows/rust.yml

```
@@ -36,7 +36,7 @@ jobs:
           override: true
 
       - name: Rust Cache
-        uses: Swatinem/rust-cache@v1.3.0
+        uses: Swatinem/rust-cache@6720f05bc48b77f96918929a9019fb2203ff71f8 # v2.0.0
```

I don't see a reason why to use a commit hash here

```suggestion
        uses: Swatinem/rust-cache@v2.0.0
```

---

As was [discussed here](https://forum.parity.io/t/github-actions-gha-versioning-updates-and-security/953/11) now we are trying to standardize GHA handling in org. Using just a tag may be not safe as tag can be easily moved to another commit. To create a fake hash much harder. Agree a bit paranoid.